### ISSUE TITLE
Projects that have Git changes have different font color

### ISSUE THREAD

**Issue by PyvesB** [AObuchow/Eclipse-Modern-Dark-Theme#55]
Hello, hello!

In the Package Explorer on my macOS machine, I noticed that Git projects that have changes appear with a slightly darker font colour. See below:
<img width="233" alt="Screenshot 2020-06-16 at 09 41 39" src="https://user-images.githubusercontent.com/10694593/84746200-e7bbb000-afb5-11ea-9b70-af5d249fd317.png">

Is this intentional? If it is, I would have expected the two colours to be inverted, in other words the projects with changes to have the lighter colour, so that they stick out more and indicate to the user that there are uncommitted changes.

Let me know if you need any more information!

**Comment by AObuchow** [comment]
Agreed the colour indications for uncommitted changes should be more obvious. 

One issue is that the git colours get set with preferences, so unfortunately they aren’t able to inherit the colours used in the theme very easily. An upstream change to egit might be needed to simplify things.

For the time being, I can hardcode new colour preferences for egit to improve this.

**Comment by AObuchow** [comment]
Upstream bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=564970

**Comment by AObuchow** [comment]
@PyvesB I'm excited to let you know that the latest experimental patch I'm working on adds some "dynamic" theming. Now, preferences such as egit's uncommitted changes foreground/background will be set to use Spectrum Theme's background/base/accent colors :D
